### ISSUE TITLE
Login problem with iOS version of Indigenous

### ISSUE THREAD

**Issue by aleksip** [swentel/indieweb#421]
I have enabled IndieAuth with the built-in authentication endpoint and Micropub.

When trying to login with the iOS version of Indigenous, I get past the Drupal login page and to the Authorize form. When I click on the Authorize button, Indigenous then gets stuck, showing the message "Authorizing with Server...". From the Drupal admin page I can see that an Authorization code for the app has been created.

Not sure if this is a problem with Indigenous or the IndieWeb modules. Has the iOS version of Indigenous been tested with Drupal?

As a side note, it seems that the iOS version is no longer maintained, and that the new apps are not yet available: https://eddiehinkle.com/2019/05/08/5/article/

**Comment by swentel** [comment]
I've never tested it with the iOS version since I don't own an iPhone, so can't really tell what's going wrong. but yes, the iOS version is no longer maintained indeed. Can you connect with another service, e.g Quill for instance?

**Comment by aleksip** [comment]
Just tried with Quill, returning from the Authorize form it shows the following error: 

```
indieauth_error
The authorization code was not able to be verified
```

So this more likely some configuration error on my site?

**Comment by swentel** [comment]
Tried signing in on Quill and works fine here, so a bit in the dark what might be wrong since there aren't that many things that can get in the way. Are there log messages by any chance on the Drupal side?

**Comment by aleksip** [comment]
Nothing in the Drupal log, so I installed a local copy of Quill and started debugging.

I discovered that the access token verification got a `301` response from my site. Once again the problem was that Drupal didn't know it was behind a TLS termination proxy, so the endpoint URLs were `http` instead of the correct `https`. I thought I had already fixed this on my site but I guess I hadn't, or something happened and the settings were lost. After adding the required settings in `settings.php` everyhing started working beautifully.

Thanks for your quick responses!
